services/generic/basic.md

```diff
@@ -1,15 +1,55 @@
-### Basic service
+# Basic Service
 
 A generic service and the most simple way to interact with a device. The actual meaning of "basic" varies from device to device.
```

Comment:
Please improve the description, eg. "most simple way to interact with a device" means nothing.
Can it be more descriptive maybe ?

Reply:
Maybe Bogus, You could use a word, "actuator".
e.g. If a device supports an actuator that is not supported by a device, then the basic service will provide a partial functionality of this unsupported actuator.

Reply:
Really good one.